RescheduleAppointments/force-app/main/default/lwc/mobileAppointmentBookingCalendar/mobileAppointmentBookingCalendar.js

```diff
@@ -1,7 +1,7 @@
 import { LightningElement, api, track } from "lwc";
 import customLabels from "./labels";
 
-export default class MobileAppointmentBookingRebookingCalendar extends LightningElement {
+export default class AaRebookingCalendar extends LightningElement {
```

Comment:
Can we rename `AaRebookingCalendar` to something more meaningful please?

Reply:
@maliroteh-sf this is accidental I reverted this change